toucan_connectors/toucan_connector.py

```diff
@@ -231,7 +231,10 @@ def get_slice(
         Exemple: if offset = 5 and limit = 10 then 10 results are expected from 6th row
         """
         df = self.get_df(data_source, permissions)
-        return DataSlice(df[offset:offset+limit], len(df))
+        if limit:
```

Comment:
```suggestion
        if limit is not None:
```
If I put `limit=0`, I expect an empty dataframe. Not all the results

Reply:
Yes thanks !